cla_public/templates/macros/cait_intervention.html

```diff
@@ -41,55 +41,61 @@ <h2 class="heading-medium">{{ survey.heading | safe }}</h2>
     }
  }
 
- {{ css }}
+ {{ css | safe }}
```

Comment:
Gives us greater flexibility. Seeing that it would require MoJDigital team access rights, anyone who hacks it would find themselves out the door pretty sharpish.